docker-vm/track.yml

```diff
@@ -18,89 +18,89 @@ icon: https://cdn.instruqt.com/assets/templates/docker.png
 tags: []
 owner: templates
 developers:
-- victor@instruqt.com
```

Comment:
was the instruqt's formatting or was it your text editor, if it's the latter I think we can omit this from the PR

Reply:
The CLI was different formatting from the vs-code extension it would seem. I think it makes sense to let the client choose how this is formatted, but for this PR, this should be reverted indeed.

Reply:
This may actually be an issue, as the track is formatted before being pushed remotely. Pulling a track will apply that formatting, even if you formatted it with your own formatter.

Reply:
yep that's why I asked we always format apply formatting on the CLI

Reply:
It would need to be done with pre-commit scripts I think. I find it a bit weird that the CLI is actually doing any formatting on its own. But perhaps there is a reason why we need to do it.